src/Module/Special/HTTPException.php

```diff
@@ -36,7 +35,10 @@ private static function getVars(\Friendica\Network\HTTPException $e)
 			500 => L10n::t('Internal Server Error'),
 			503 => L10n::t('Service Unavailable'),
 		];
-		$title = defaults($titles, $e->getCode(), 'Error ' . $e->getCode());
+		$title = (empty($e->httpdesc) ?
```

Comment:
This is never empty, which means the exception title will never appear translated for non-custom `httpdesc`.